appinventor/components/src/com/google/appinventor/components/runtime/Form.java

```diff
@@ -1971,6 +1971,31 @@ public void BlocksToolkit(String json) {
     // project properties file
   }
 
+  /**
+   * Gets the name of the underlying platform running the app. Currently, this is the text
+   * "Android". Other platforms may be suppoorted in the future.
```

Comment:
```suggestion
   * "Android". Other platforms may be supported in the future.
```

Minor typo here and in the documentation.

Reply:
@conorshipp Thanks for catching that. I've fixed it.